examples/nextjs-auth/pages/api/graphql.ts

```diff
@@ -1,64 +1,28 @@
-// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
-import { createYoga, createSchema } from 'graphql-yoga'
-
+import { createServer } from '@graphql-yoga/node'
```

Comment:
Can't we make this with v3? This import belongs to v2 I guess.

Reply:
Effectively ,
I did a bad import... I have to keep  import { createYoga, createSchema } from 'graphql-yoga' and remove import { createServer } from '@graphql-yoga/node'